quantum/process_keycode/process_unicode_common.h

```diff
@@ -42,12 +42,12 @@ void unicode_input_start(void);
 void unicode_input_finish(void);
 void register_hex(uint16_t hex);
 void send_unicode_hex_string(const char *str);
-bool process_record_unicode_common(uint16_t keycode, keyrecord_t *record);
+bool process_unicode_common(uint16_t keycode, keyrecord_t *record);
 
 #define UC_OSX 0  // Mac OS X
 #define UC_LNX 1  // Linux
-#define UC_WIN 2  // Windows 'HexNumpad'
-#define UC_BSD 3  // BSD (not implemented)
+#define UC_BSD 2  // BSD (not implemented)
+#define UC_WIN 3  // Windows 'HexNumpad'
```

Comment:
It just makes more sense this way, and is in line with the order you naturally declared the songs and stuff in.

Reply:
I'd rather preserve the existing order, and re-order my additions.  

Which I've done.